### DISCUSSION TITLE
Are there any single stroke fonts that can be used to render dxf files using ezdxf

### DISCUSSION THREAD

**Discussion by joejoseph007** [mozman/ezdxf#970] [Q&A]
I know that **arialuni** encapsulates a lot of characters/glyphs, but are there any similar (in terms of number of characters/glyphs) that are single-stroke fonts like Autocad's Simplex font?

**Comment by mozman** [comment]
There exist a `bigfont` attribute in the `STYLE` entity to support languages with a large character set. _Big fonts_ have a slightly different file format than the usual SHX fonts. I haven't used _big fonts_ and I don't know how they work. I don't know if the encoding corresponds to Unicode. There is no support for _big fonts_ in `ezdxf`. If you use _big fonts_ - `ezdxf` can't render the font.

**Reply by joejoseph007** [reply]
Thanks for your reply! I did want to know if single stoke fonts can be rendered?

**Reply by mozman** [reply]
Sorry I don't understand what you want to achieve.
